### PR TITLE
Password recovery

### PR DESCRIPTION
**Pull request by taras0024**

### CODE REVIEW

File: library/catalog/views.py [hunk 1]
```diff
@@ -309,3 +312,39 @@ def collections_page(request):
...
     return render(request, 'authors.html', {'authors': authors})
+
+
+def reset_password(request):
+    if request.method == 'POST':
+        email = request.POST.get('email')
+        user = MongoUser.objects.filter(email=email).first()
+        if user:
+            new_password = ''
+            for i in range(5):
```
Comment (ivan-dubei) [inline library/catalog/views.py]: Antipattern - magic numbers

File: library/catalog/views.py [hunk 2]
```diff
@@ -309,3 +312,39 @@ def collections_page(request):
...
+                new_password += random.choice(string.ascii_letters)
+                new_password += str(random.choice(range(1, 21)))
```
Comment (ivan-dubei) [inline library/catalog/views.py]: Antipattern - magic numbers

File: library/catalog/views.py [hunk 3]
```diff
@@ -309,3 +312,41 @@ def collections_page(request):
...
+
+def reset_password(request):
+    if request.method == 'POST':
+        email = request.POST.get('email')
+        user = MongoUser.objects.filter(email=email).first()
+        if user:
+            new_password = ''
+            numbers_list = [str(i) for i in range(21)]
+            eng_alph = string.ascii_letters
+            for i in range(5):
```
Comment (ivan-dubei) [inline library/catalog/views.py]: PASSWORD_LENGTH = 5
for i in range(PASSWORD_LENGTH ):

File: library/catalog/views.py [hunk 4]
```diff
@@ -309,3 +312,41 @@ def collections_page(request):
...
     return render(request, 'authors.html', {'authors': authors})
+
+
+def reset_password(request):
+    if request.method == 'POST':
+        email = request.POST.get('email')
+        user = MongoUser.objects.filter(email=email).first()
+        if user:
+            new_password = ''
+            numbers_list = [str(i) for i in range(21)]
```
Comment (ivan-dubei) [inline library/catalog/views.py]: MAX_PASSWORD_NUM = 21
numbers_list = [str(i) for i in range(MAX_PASSWORD_NUM )]

Don't forget to put constants to the top of the module, after the imports

**Review by AndriiSkavreniuk — APPROVED** [review]

**Review by ivan-dubei — APPROVED** [review]

**Pull request merged**